pubsub/redis/redis.go

```diff
@@ -129,6 +134,46 @@ func parseRedisMetadata(meta pubsub.Metadata) (metadata, error) {
 		m.concurrency = uint(concurrency)
 	}
 
+	if val, ok := meta.Properties[dialTimeoutInSec]; ok && val != "" {
```

Comment:
Note how `redeliverInterval` and `processingTimeout` are parsed. First as uint milliseconds (e.g. `5000`), then as a duration (e.g. `5s`).

Reply:
Thanks for advice, I will Parse duration-based settings like how redeliverInterval and processingTimeout are parsed.